### DISCUSSION TITLE
how to run a Nuxt app in a sub directory in a nginx  ?

### DISCUSSION THREAD

**Discussion by BhanuPrakash056** [nuxt/nuxt#23306] [Questions]
I tried using the below code

export default defineNuxtConfig({
  build: {
    baseURL: '/app_name',
  }
})

**Comment by KeJunMao** [comment]
```ts
// nuxt.config.ts
export default defineNuxtConfig({
  app: {
    baseURL: '/app_name/',
  },
})
```

see [useRuntimeConfig - app.baseURL](https://nuxt.com/docs/api/composables/use-runtime-config#appbaseurl)
